.gitignore

```diff
@@ -41,6 +41,12 @@ build/
 .project
 .classpath
 
+# p8e files
+ContractHash*.kt
+ProtoHash*.kt
```

Comment:
We can refine this later if it would exclude new files that we do want to be committed